pyrogram/client/client.py

```diff
@@ -1788,7 +1788,7 @@ def save_file(
 
         part_size = 512 * 1024
 
-        if isinstance(path, str):
+        if isinstance(path, str) or isinstance(path, PurePath):
```

Comment:
Use `isinstance(path, (str, PurePath))` instead. The function allows multiple types to be checked with a single call.